requirements/worker.txt

```diff
@@ -1,5 +1,6 @@
 matplotlib==3.3.3
 networkx==2.5
+numpy==1.21.0
```

Comment:
Do you know what is the error when upgrading it through `__init__.py`

Reply:
Due to existing numpy version the module reload is not automatically working and the force `reload` call fails with this error https://cloudcv.slack.com/archives/C2P84STC6/p1653838354324559